openhtf/plugs/usb/adb_protocol.py

```diff
@@ -79,7 +79,7 @@
 import collections
 import itertools
 import logging
-import Queue
+import queue
```

Comment:
not PY2 compatible

either version check or do a try-except to `import Queue as queue`